packages/graphql/src/schema/query.ts

```diff
@@ -271,7 +295,8 @@ export const Query = builder.queryType({
           return CourseService.getCourseData(args, ctx)
         },
       }),
-      liveSession: t.field({
+
+      liveSession: asUser.field({
```

Comment:
Changed `asAuthenticated` to `asUser` as this query is only used in `frontend-manage` and `frontend-control`